internal/awsservice/s3.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package awsservice
+
+import (
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+
+func DownloadFile(bucket, key, outFilename string) error {
```

Comment:
DownloadFile is basically the same code that was already in the CanaryTest. I just made it generic and moved it to this `s3.go` file.